tools/StaticAnalysis/Exceptions/Az.Network/BreakingChangeIssues.csv

```diff
@@ -25,3 +25,6 @@
 "Az.Network","Microsoft.Azure.Commands.Network.GetAzurePrivateEndpointConnection","Get-AzPrivateEndpointConnection","0","1050","The parameter set 'ByResource' for cmdlet 'Get-AzPrivateEndpointConnection' has been removed.","Add parameter set 'ByResource' back to cmdlet 'Get-AzPrivateEndpointConnection'."
 "Az.Network","Microsoft.Azure.Commands.Network.RemoveAzurePrivateEndpointConnection","Remove-AzPrivateEndpointConnection","0","1050","The parameter set 'ByResource' for cmdlet 'Remove-AzPrivateEndpointConnection' has been removed.","Add parameter set 'ByResource' back to cmdlet 'Remove-AzPrivateEndpointConnection'."
 "Az.Network","Microsoft.Azure.Commands.Network.SetAzurePrivateEndpointConnection","Set-AzPrivateEndpointConnection","0","1050","The parameter set 'ByResource' for cmdlet 'Set-AzPrivateEndpointConnection' has been removed.","Add parameter set 'ByResource' back to cmdlet 'Set-AzPrivateEndpointConnection'."
+"Az.Network","Microsoft.Azure.Commands.Network.GetAzureFirewallCommand","Get-AzFirewall","0","3010","The property 'publicIPAddresses' of type 'Microsoft.Azure.Commands.Network.Models.PSAzureFirewallHubIpAddresses' has been removed.","Add the property 'publicIPAddresses' back to type 'Microsoft.Azure.Commands.Network.Models.PSAzureFirewallHubIpAddresses'."
+"Az.Network","Microsoft.Azure.Commands.Network.NewAzureFirewallCommand","New-AzFirewall","0","3010","The property 'publicIPAddresses' of type 'Microsoft.Azure.Commands.Network.Models.PSAzureFirewallHubIpAddresses' has been removed.","Add the property 'publicIPAddresses' back to type 'Microsoft.Azure.Commands.Network.Models.PSAzureFirewallHubIpAddresses'."
+"Az.Network","Microsoft.Azure.Commands.Network.NewAzureFirewallHubIpAddress","New-AzFirewallHubIpAddress","0","3010","The property 'publicIPAddresses' of type 'Microsoft.Azure.Commands.Network.Models.PSAzureFirewallHubIpAddresses' has been removed.","Add the property 'publicIPAddresses' back to type 'Microsoft.Azure.Commands.Network.Models.PSAzureFirewallHubIpAddresses'."
```

Comment:
Please remove the preannounce in last [release](https://github.com/Azure/azure-powershell/pull/19363).

Reply:
@wyunchi-ms I didn't get what you wanted me to remove? Can you please specify what I need to remove?

Reply:
We added these preannounce last release to notify user there will be some breaking changes in this release.
But now we are doing the breaking changes, we need to remove the notifications. Or users will still get those notifications in next release.

Reply:
@Gizachew-Eshetie

Reply:
@wyunchi-ms Done.